AI/OpenAI/Whisper/README.md

```diff
@@ -0,0 +1,81 @@
+# OpenAI Whisper
+
+In this document, you will find how to install and run a simple Whisper example.
+
+## What is Whisper?
+
+See Whisper [website](https://openai.com/index/whisper/) and
+[documentation](https://platform.openai.com/docs/guides/speech-to-text).
+
+## Security
+
+Please, **carefully** read Harvard's [AI
+guidelines](https://huit.harvard.edu/ai/guidelines) and [Generative AI
+tool comparision](https://huit.harvard.edu/ai/tools).
+
+You can only use openAI and other genAI on non-sensitive (data
+security level 1) public data on Cannon. See [FASRC
+Guidelines](https://docs.rc.fas.harvard.edu/kb/openai/).
+
+For data security ***levels 2 and 3***, you need to work with your
+school to discuss your needs. It is your responsibility to make sure
+you get setup with the appropriate contractual coverage and
+environment -- esp. to avoid having the model learn from your input
+and leak sensitive information.
+
+## Installation
+
+You can install Whisper in a conda/mamba environment. Assuming that
+you have already created and installed OpenAI as described
+[here](https://github.com/fasrc/User_Codes/tree/master/AI/OpenAI),
+Whisper can be easily installed as mentioned below:
+
+```bash
+[jharvard@boslogin01 ~]$ salloc --partition test --time 01:00:00 --mem-per-cpu 4G -c 2
+[jharvard@holy8a24301 ~]$ module load python
+[jharvard@holy8a24301 ~]$ export PYTHONNOUSERSITE=yes
+[jharvard@holy8a24301 ~]$ source activate openai_env
+[jharvard@holy8a24301 ~]$ pip install -U openai-whisper
+[jharvard@holy8a24301 ~]$ conda install conda-forge::ffmpeg -y
+[jharvard@holy8a24301 ~]$ conda install conda-forge::rust -y
+[jharvard@holy8a24301 ~]$ conda deactivate
+```
+
+## Run OpenAI
+
+As mentioned
+[here](https://github.com/fasrc/User_Codes/tree/master/AI/OpenAI), you
+will need to provide an OpenAI key. If you haven't already, you can
+generate one from
+[https://platform.openai.com/api-keys](https://platform.openai.com/api-keys). It
+is advisable to store this key in a safe location, like your
+`~/.bashrc` file on Cannon. Additionally, you can also store the
+`SSL_CERT_FILE` in your `~/.bashrc` instead of exporting it every time
+you want to run OpenAI or Whisper.
+
+```bash
+# Request an interactive job
+[jharvard@boslogin01 ~]$ salloc --partition test --time 01:00:00 --mem-per-cpu 4G -c 2
```

Comment:
I just thought that since we are doing openai stuff, should we request a gpu_test with --grep=gpu:1? I know openai example also has test partition, so we would have to change there too. The test script `openai-whisper-test.py` worked on both test and gpu_test partitions.

Reply:
done